src/trimStart.ts

```diff
@@ -9,11 +9,12 @@ const _trimStart = require('lodash/trimStart');
  * trimStart([1, 2, 3], [999, 2]) === [1, 2, 3] since source[0] does not equal elems[0]
  */
 export const trimStart = (target: any[] | string, elems: any[] | string) => {
-  if (typeof target === 'string') {
+  if (typeof target === 'string' && typeof elems === 'string') {
     return _trimStart(target, elems);
   }
 
-  if (!elems || !elems.length || !(elems instanceof Array)) return target;
+  if (!target || !Array.isArray(target) || !target.length || !elems || !Array.isArray(elems) || !elems.length)
```

Comment:
Is this duck-type like change driven by issues with typings or does it actually address some issue?
Just asking.

Reply:
We were not handling it before because it wasn't called out by the typings (since we were using require, which clobbers all the typings). This just fixes it up to match the typings.